README_VERITABLE.md

```diff
@@ -43,21 +43,21 @@ The project aims to enable supply chains to share insights and data across multi
 
 The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI (http://localhost:3000/swagger) that is provided with the server. The OpenAPI spec can be viewed on the `/api-docs` endpoint (e.g. http://localhost:3000/api-docs).
 
-Bellow you will find commands for starting up the containers in docker (see the [Using Docker](#using-docker-easiest) section) or via cli (see [Via Cli](#via-cli)).
+Bellow you will find commands for starting up the containers in docker (see the [Using Docker](#using-docker-easiest) section).
 
 > The OpenAPI spec is generated from the model classes used by Aries Framework JavaScript. Due to limitations in the inspection of these classes, the generated schema does not always exactly match the expected format. Keep this in mind when using this package. If you encounter any issues, feel free to open an issue.
 
 > In case `npm i` fails with the references to the `node-gyp`. Please install xcode utilities by ruuning -> `brew install xcode`. Before executing `npm i` delete your node_modules in case it has some old references `rm -rf /node_modules`
 
-### Rest Client Args
+### Rest Client Envs
 
-The CLI args are defined under `src > cli.ts ` They are used to start up a service.
+The Envs are defined under `src > env.ts ` They are used to start up a container.
 | Argument added | Required | Default | Description |
 | :------------- | :------: | :------ | :------- |
-| ipfs-origin | Y | http://localhost:5001 | IPFS endpoint |
-|opa-origin |N |http://localhost:8181 |OPA endpoint |
-|persona-title | N |"Veritable Cloudagent"|Tab name which you can see in your browser |
-|persona-color |N |"white" |Defines the background colour of swagger documentation|
+| IPFS_ORIGIN | Y | http://localhost:5001 | IPFS endpoint |
+|OPA_ORIGIN |N |http://localhost:8181 |OPA endpoint |
+|PERSONA_TITLE | N |"Veritable Cloudagent"|Tab name which you can see in your browser |
+|PERSONA_COLOR |N |"white" |Defines the background colour of swagger documentation|
```

Comment:
We need to document the envs better here I think

Reply:
would you want all envs documented?

Reply:
yes please